src/App.tsx

```diff
@@ -14,11 +14,124 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGoods: string[];
+  count: number,
```

Comment:
Do you really need this variable? You can always access selectedGoods.length, that has the same value